packages/cli/test/unit/License.test.ts

```diff
@@ -71,9 +71,9 @@ describe('License', () => {
 	});
 
 	test('check fetching feature values', async () => {
-		await license.getFeatureValue(MOCK_FEATURE_FLAG, false);
+		await license.getFeatureValue(MOCK_FEATURE_FLAG);
 
-		expect(LicenseManager.prototype.getFeatureValue).toHaveBeenCalledWith(MOCK_FEATURE_FLAG, false);
+		expect(LicenseManager.prototype.getFeatureValue).toHaveBeenCalledWith(MOCK_FEATURE_FLAG);
```

Comment:
We don't want to check the return value as well?

Reply:
I didn't do any changes to this tests, just removed the second parameter as it's no longer user